instrumentation/grizzly-client-1.9/src/main/java/io/opentelemetry/instrumentation/auto/grizzly/client/GrizzlyClientTracer.java

```diff
@@ -16,14 +16,16 @@
 
 package io.opentelemetry.instrumentation.auto.grizzly.client;
 
+import com.ning.http.client.FluentCaseInsensitiveStringsMap;
 import com.ning.http.client.Request;
 import com.ning.http.client.Response;
 import io.opentelemetry.context.propagation.HttpTextFormat.Setter;
 import io.opentelemetry.instrumentation.api.tracer.HttpClientTracer;
 import java.net.URI;
 import java.net.URISyntaxException;
 
-public class GrizzlyClientTracer extends HttpClientTracer<Request, Response> {
+public class GrizzlyClientTracer
+    extends HttpClientTracer<Request, FluentCaseInsensitiveStringsMap, Response> {
```

Comment:
Are you sure it wouldn't be better `HttpClientTracer<Request, Request, Response>`?

Reply:
`FluentCaseInsensitiveStringsMap` most closely represents the `HEADERS` type, but if you still prefer `Request` I don't mind and will change it

Reply:
Yeah, my comments probably actually mean "are you sure that HEADERS is the best candidate for this new type parameter"? I have review around half of the changed files and it seems to me that Request is actually enough for like 90%+ cases. Even if actually need a separate type parameter for those 10%, it does not make sense to force that HEADERS name to the letter and use such strange types.

Rename HEADER type to [Carrier](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/context/api-propagators.md#carrier) and use the simplest/most natural type?

Reply:
oh yes I like renaming `HEADERS` type to `CARRIER` will do

Reply:
done